cmd/channel/rocketchat/main.go

```diff
@@ -23,7 +23,7 @@ func main() {
 
 func (ch *RocketChat) SendNotification(req *plugin.NotificationRequest) error {
 	var output bytes.Buffer
-	_, _ = fmt.Fprintf(&output, "[#%d] %s %s is %s\n\n", req.Incident.Id, req.Event.Type, req.Object.Name, req.Incident.Severity)
+	_, _ = fmt.Fprint(&output, plugin.FormatSubject(req))
```

Comment:
Thanks for [your changes](https://github.com/Icinga/icinga-notifications/compare/fd02071db871792456d9a5e02a49f2a0bb2d0019..fb863216e971e886b2d23893490a1466cf31af7c). But now this one misses newlines :upside_down_face:

Reply:
Duplicate of <https://github.com/Icinga/icinga-notifications/pull/146#discussion_r1576458904>, @julianbrost was a bit faster.